### PR TITLE
🏺Upgrade Olympia types

### PR DESCRIPTION
**Pull request by jodator**
See #743

### CODE REVIEW

File: packages/ui/src/common/api/schemas/schema.graphql [hunk 1]
```diff
@@ -46,8 +38,8 @@ type ApplicationFormQuestionAnswerConnection {
 }
 
 input ApplicationFormQuestionAnswerCreateInput {
-  applicationId: ID!
-  questionId: ID!
+  application: ID!
```
Comment (DevNikita16) [inline packages/ui/src/common/api/schemas/schema.graphql:41]: Is there any reason not to use `applicationId` or `questionId`? For me `...Id` represents ID, but without ID is object

File: packages/ui/src/mocks/data/seedWorkers.ts [hunk 2]
```diff
@@ -37,11 +37,11 @@ export const seedWorker = (worker: RawWorker, server: any) => {
 const seedWorkerStatus = (status: WorkerStatus, id: string, server: any) => {
   switch (status) {
     case 'active':
-      return server.schema.create('WorkerStatusActive', {})
+      return server.schema.create('WorkerStatusActive', { phantom: 0 })
```
Comment (DevNikita16) [inline packages/ui/src/mocks/data/seedWorkers.ts:39]: I think I can use this in my new commit instead of my resolvers, I should just filter by phantom:0

**Review by DevNikita16 — COMMENTED** [review]
Like this changes 👍 My only question why you've renamed `applicationId` to `application` etc.

File: packages/ui/src/common/api/schemas/schema.graphql (hunk 1 above)
Comment (jodator) [reply packages/ui/src/common/api/schemas/schema.graphql:41]: The schema is defined externally. Unfortunately, we have to live with that as we're the consumers here :)

File: packages/ui/src/mocks/data/seedWorkers.ts (hunk 2 above)
Comment (jodator) [reply packages/ui/src/mocks/data/seedWorkers.ts:39]: I think that filtering by `__typename: 'WorkerStatusActive'` will be more future-proof. The `phantom` was used to overcome Hydra schema shortcomings AFAIR.

Comment (DevNikita16) [reply packages/ui/src/mocks/data/seedWorkers.ts:39]: https://dzhelezov.gitbook.io/hydra/docs/schema-spec/variant-types
I was not able to filter by __typename, but maybe will be able to intoduce isTypeOf after you changes for nested filtering

**Review by p-sad — APPROVED** [review]

**Pull request merged**